Sources/Account/PublicAddress.swift

```diff
@@ -58,6 +58,8 @@ public struct PublicAddress {
 
     public var fogReportUrlString: String? { fogInfo?.reportUrlString }
 
+    public var addressHash: String? { calculateAddressHash()?.hex }
```

Comment:
Let's name this to reflect that it's hex-encoded. We might want a deserialized or base64-encoded variant, to match the Android SDK. 
```suggestion
    public var addressHashHex: String? { calculateAddressHash()?.hex }
```

Reply:
changed to a `Data` object then you can choose hex or base64 like below:

```swift
publicAddress.addressHash?.hexEncodedString()
publicAddress.addressHash?.base64EncodedString()
```

Reply:
Beautiful, thank you!